### PR TITLE
[bitnami/mariadb-galera] Allow extraFlags on Mariadb metrics container

### PR DESCRIPTION
**Pull request by qunayu**
**Description of the change**

Allows users to define extra flags to use for the MariaDB metrics exporter.

**Benefits**

Users can now specify extra flags for the mysqld_exporter tool to use. This will, for example, allow users to control what metrics they want scraped from their db instance.

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

None

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb-galera/values-production.yaml [hunk 1]
```diff
@@ -555,6 +555,11 @@ metrics:
     ##
     # pullSecrets:
     #   - myRegistryKeySecretName
+  ## MySQL exporter additional command line flags
+  ## Can be used to specify command line flags, for example:
+  ##
+  ## extraFlags: "--collect.binlog_size"
+  ##
```
Comment (juan131) [inline bitnami/mariadb-galera/values-production.yaml:562]: Could we use an array for this? IMHO it's easier to manage flags as an array:
```suggestion
  ## MySQL exporter additional command line flags
  ## Can be used to specify command line flags
  ## E.g.:
  ## extraFlags:
  ##   --collect.binlog_size
  ## 
  extraFlags: []
```

File: bitnami/mariadb-galera/templates/statefulset.yaml [hunk 2]
```diff
@@ -242,6 +242,8 @@ spec:
           image: {{ template "mariadb-galera.metrics.image" . }}
           imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
           env:
+            - name: MARIADB_METRICS_EXTRA_FLAGS
+              value: {{ default "" .Values.metrics.extraFlags | quote }}
```
Comment (juan131) [inline bitnami/mariadb-galera/templates/statefulset.yaml:246]: If you treat `.Values.metrics.extraFlags` as an array, use:
```suggestion
            - name: MARIADB_METRICS_EXTRA_FLAGS
              value: {{ default "" (join " " .Values.metrics.extraFlags) | quote }}
```

File: bitnami/mariadb-galera/values.yaml [hunk 3]
```diff
@@ -554,6 +554,11 @@ metrics:
     ##
     # pullSecrets:
     #   - myRegistryKeySecretName
+  ## MySQL exporter additional command line flags
+  ## Can be used to specify command line flags, for example:
+  ##
+  ## extraFlags: "--collect.binlog_size"
+  ##
```
Comment (juan131) [inline bitnami/mariadb-galera/values.yaml:561]: Same here:
```suggestion
  ## MySQL exporter additional command line flags
  ## Can be used to specify command line flags
  ## E.g.:
  ## extraFlags:
  ##   --collect.binlog_size
  ## 
  extraFlags: []
```

**Review by juan131 — COMMENTED** [review]
Thanks so much for your contribution!! Please take a look to my suggestions and let me know what you think

File: bitnami/mariadb-galera/values-production.yaml (hunk 1 above)
Comment (qunayu) [reply bitnami/mariadb-galera/values-production.yaml:562]: Yup, good idea, the changes look good to me! I'll add a small change to the README too.

File: bitnami/mariadb-galera/README.md [hunk 4]
```diff
@@ -166,6 +166,7 @@ The following table lists the configurable parameters of the MariaDB Galera char
 | `metrics.image.repository`           | MariaDB Prometheus exporter image name                                                                                                                      | `bitnami/mysqld-exporter`                                         |
 | `metrics.image.tag`                  | MariaDB Prometheus exporter image tag                                                                                                                       | `{TAG_NAME}`                                                      |
 | `metrics.image.pullPolicy`           | MariaDB Prometheus exporter image pull policy                                                                                                               | `IfNotPresent`                                                    |
+| `metrics.extraFlags`                 | MariaDB Prometheus exporter additional command line flags                                                                                                   | `nil`                                                             |
```
Comment (qunayu) [inline bitnami/mariadb-galera/README.md:169]:
```suggestion
| `metrics.extraFlags`                 | MariaDB Prometheus exporter additional command line flags                                                                                                   | `[]`                                                              |
```

**Review by juan131 — APPROVED** [review]
Thanks for addressing the suggestions! LGTM

**Pull request merged**